### PR TITLE
Update aliases.asciidoc

### PR DESCRIPTION
**Pull request by emirb**
Some uppercase cleanup.

### CODE REVIEW

File: docs/reference/indices/aliases.asciidoc [hunk 1]
```diff
@@ -71,7 +71,7 @@ POST /_aliases
 // CONSOLE
 // TEST[s/^/PUT test1\nPUT test2\n/]
 
-Multiple indices can be specified for an action with the `indices` array syntax:
+Multiple indices for an action can be specified with the `indices` array syntax:
```
Comment (nik9000) [inline docs/reference/indices/aliases.asciidoc]: I like the previous wording better for this

**Comment by nik9000** [comment]
I left a change request.

Can you sign the CLA so we can merge this when we've finished with the review? We need it for all contributions.

**Comment by jasontedor** [comment]
@emirb Are you interested in taking this PR to the finish line?

**Comment by emirb** [comment]
@nik9000 @jasontedor Signed upon opening the pull request :)

**Comment by nik9000** [comment]
Thanks for fixing this up @emirb! Sorry I didn't notice until now. I'll merged and cherry-pick as appropriate.

**Comment by nik9000** [comment]
There we go! All merged and cherry-picked. Thanks again @emirb!

**Pull request merged**